ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,224 @@
+}
+
+const triggerSet: TriggerSet<Data> = {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  zoneId: ZoneId.AsphodelosTheThirdCircle,
+  timelineFile: 'p3n.txt',
+  triggers: [
+    {
+      id: 'P3N Experimental Fireplume 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'P3N Experimental Fireplume 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.outOfMiddle!();
+      },
+      outputStrings: {
+        outOfMiddle: {
+          en: 'Out Of Middle',
+          de: 'Raus aus der Mitte',
+          ja: '横へ',
+          cn: '远离中间',
+          ko: '가운데 피하기',
+        },
+      },
+    },
+    {
+      id: 'P3N Scorched Exaltation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B8', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Heat of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B2', source: 'Phoinix' }),
+      suppressSeconds: 1,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'P3N Darkened Fire Aoe',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: ['010[C-F]'] }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => {
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Go to add',
+        },
+      },
+    },
+    {
+      id: 'P3N Right Cinderwing',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B4', source: 'Phoinix', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'P3N Left Cinderwing',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B5', source: 'Phoinix', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      // Could check the log line's x y coordinate to determine from where to where it charges, npc at charge target casts 66AF?
+      id: 'P3N Trail of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66AD', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.avoidCharge!();
+      },
+      outputStrings: {
+        avoidCharge: {
+          en: 'Avoid Charge',
+          de: 'Charge ausweichen',
+          fr: 'Évitez les charges',
+          ja: '突進避けて',
+          cn: '躲避冲锋',
+          ko: '돌진을 피하세요',
+        },
+      },
+    },
+    {
+      id: 'P3N Sunbird Spawn',
+      type: 'AddedCombatant',
+      netRegex: NetRegexes.addedCombatant({ name: 'Sunbird', capture: false }),
+      suppressSeconds: 1,
+      response: Responses.killAdds(),
```

Comment:
I did an explaning callout for tanks in e32226eea0497fd9f12913a9d7136930d24c0a28 (because who else is going to split the adds realistically?) is that one ok?

Reply:
in t12 there was just a callout that the add is comming soon without further information.
It should be a "Kill adds without overlapping dead bodies" so 1 could be dead in the center and dont actually have to be on the edge

Reply:
Pretty sure it's not about the actual models stacking but about the circles not intersecting, I could be wrong though the one time I wiped to this they were stacked on point.